src/actions/remove.rs

```diff
@@ -1,7 +1,90 @@
+use crate::actions::remove;
 use crate::args::RemoveArgs;
 use crate::config::UserConfig;
+use std::io::{stdout, Write, stdin};
+use mprs::utils;
+use mprs::utils::{print_table, list_dir};
+use std::fs::remove_file;
+use std::io;
+use std::path::Path;
+use prettytable::{Table, Cell, Row};  // should this be in utils?
```

Comment:
Yes it should

Reply:
Yeah, I had these imported 'cause I thought I'd have to build the table myself. Turns out I figured out how to use `utils::print_table` with what I had.

Reply:
So remove just line 10, right?